pkg/hubble/exporter/exporter.go

```diff
@@ -139,6 +140,20 @@ func (e *exporter) OnDecodedEvent(ctx context.Context, ev *v1.Event) (bool, erro
 	if !filters.Apply(e.opts.AllowFilters(), e.opts.DenyFilters(), ev) {
 		return false, nil
 	}
+
+	// Process OnExportEvent hooks
+	for _, f := range e.opts.OnExportEvent {
+		stop, err := f.OnExportEvent(ctx, ev, e.encoder)
+		if err != nil {
+			e.logger.WithError(err).WithField("event", ev).Info("failed in OnExportEvent")
```

Comment:
Generally, we don't want to log the full event above the Debug log level. First and foremost to avoid overloading logging but also there is sometimes a privacy concern (here I think it's fine because the event was meant to be exported anyway, and thus users should already be cautious wrt sensitive data, i.e. configure hubble redaction accordingly). cc @chancez what do you think?

Reply:
Good points. I actually replicated the same hook processing logic from `LocalObserverServer.Start()`. Happy to make changes / align them if we feel like doing that.

Reply:
I agree with Alex, and this should be an `Error` or `Warn` instead of `Info`. Additionally the log could probably phrased `OnExportEvent failed`.

Reply:
So remove the event field completely or log one/subset of fields? And should we also update `LocalObserverServer.Start()` to reflect that as well ?

Reply:
removed event logging and updated to Warn level